### PR TITLE
bestie: Remove Cloud Run support

### PR DESCRIPTION
**Pull request by minor-fixes**
This change modifies bestie to prepare it to run in k8s instead of Cloud
Run:
* Server is changed to the "normal" server rather than the one
  specifically for Cloud Run (which needed a different setup for
  proper HTTP/2 handling)
* Bazel rule to deploy to Cloud Run is removed

This change also adds Greg as a codeowner of bestie.

Tested:
* brought up server locally with `bazel run //bestie/server:bestie`
* Metrics endpoint is alive: `curl localhost:6433/metrics`
* Prints BES messages when pointing bazel at it with `bazel build
  //bestie/... --bes_backend=grpc://localhost:6433`

Jira: INFRA-436

### CODE REVIEW

**Comment by minor-fixes** [comment]
@msgoldflam How/why are you the sole CODEOWNER on this review? (I don't see a CODEOWNER file, and don't have permissions to inspect repository configuration)

**Review by gpaussaenf — APPROVED** [review]

**Comment by gpaussaenf** [comment]
I previously added Mike's GitHub ID to the existing enkit/.github/CODEOWNERS file for the /bestie/ code path. Your ID was already in there. Mike wanted to be notified of any changes to the BES Endpoint code.

**Comment by minor-fixes** [comment]
> I previously added Mike's GitHub ID to the existing enkit/.github/CODEOWNERS file for the /bestie/ code path

Gotcha; I wasn't looking for the file in the right place!

Adding you to CODEOWNERS as well so that we're not blocked on Mike

**Review by msgoldflam — APPROVED** [review]
Discussed with Greg so approving

**Pull request merged**
